AjobK/Athena
# 64 api

closes #64 

 **What is it supposed to do**
You should be able to fetch data from the API

 **How can we test your branch**
Install insomnia import the insomnia.json file (`system/Insomnia`) Then all the router will be available.
Make sure you have XAMMP running with the database (`system/database`)
Then run `npm ci`
Then run `npm run serve`

 **What broweser have been checked**
No browser check needed, because it is backend
